recipe/meta.yaml

```diff
@@ -34,10 +36,64 @@ test:
   imports:
     - ax
     - ax.benchmark
+    - ax.benchmark.botorch_modular
+    - ax.benchmark.tests
+    - ax.core
+    - ax.core.tests
+    - ax.early_stopping
+    - ax.early_stopping.strategies
+    - ax.early_stopping.tests
+    - ax.exceptions
+    - ax.metrics
+    - ax.metrics.tests
+    - ax.modelbridge
+    - ax.modelbridge.strategies
+    - ax.modelbridge.tests
+    - ax.modelbridge.transforms
+    - ax.models
+    - ax.models.discrete
+    - ax.models.numpy
+    - ax.models.random
+    - ax.models.tests
+    - ax.models.torch
+    - ax.models.torch.botorch_modular
+    - ax.models.torch.tests
+    - ax.plot
+    - ax.plot.css
+    - ax.plot.js
+    - ax.plot.js.common
+    - ax.plot.tests
+    - ax.runners
+    - ax.service
+    - ax.service.tests
+    - ax.service.utils
+    - ax.storage
+    - ax.storage.json_store
+    - ax.storage.json_store.tests
+    - ax.storage.sqa_store
+    - ax.storage.sqa_store.tests
+    - ax.utils
+    - ax.utils.common
+    - ax.utils.common.tests
+    - ax.utils.flake8_plugins
+    - ax.utils.measurement
+    - ax.utils.measurement.tests
+    - ax.utils.notebook
+    - ax.utils.report
+    - ax.utils.report.resources
+    - ax.utils.report.tests
+    - ax.utils.stats
+    - ax.utils.stats.tests
+    - ax.utils.testing
+    - ax.utils.testing.metrics
+    - ax.utils.testing.tests
+    - ax.utils.tutorials
   commands:
     - pip check
+    - pytest -vvv --capture=tee-sys --pyargs ax
   requires:
     - pip
+    - pytest
```

Comment:
You'll likely also want to add a bunch of other test dependencies.
https://github.com/facebook/Ax/blob/main/setup.py#L33

Developers aren't always too good at distinguishing these from development dependencies.

Or maybe just skip the `pytest` command

Reply:
yeah I guess i can skip it for now